packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -195,43 +195,33 @@ Use `shorebird flutter versions list` to list available versions.
         final appId = shorebirdYaml.getAppId(flavor: flavor);
         final app = await codePushClientWrapper.getApp(appId: appId);
 
-        final bundleDirPath = p.join(
-          projectRoot.path,
-          'build',
-          'app',
-          'outputs',
-          'bundle',
-        );
-
-        final apkDirPath = p.join(
-          projectRoot.path,
-          'build',
-          'app',
-          'outputs',
-          'apk',
-        );
-        final bundlePath = flavor != null
-            ? p.join(
-                bundleDirPath,
-                '${flavor}Release',
-                'app-$flavor-release.aab',
-              )
-            : p.join(bundleDirPath, 'release', 'app-release.aab');
-        final apkPath = flavor != null
-            ? p.join(
-                apkDirPath,
-                flavor,
-                'release',
-                'app-$flavor-release.apk',
-              )
-            : p.join(apkDirPath, 'release', 'app-release.apk');
+        final File apkFile;
+        final File bundleFile;
+        try {
+          bundleFile = shorebirdAndroidArtifacts.findAppBundle(
+            project: projectRoot,
+            flavor: flavor,
+          );
```

Comment:
```suggestion
        final File apkFile;
        final File aabFile;
        try {
          aabFile = shorebirdAndroidArtifacts.findAab(
            project: projectRoot,
            flavor: flavor,
          );
```

Minor but for consistency might want to rename?